pkg/target/target.go

```diff
@@ -92,10 +93,42 @@ func (h *K8sValidationTarget) HandleReview(obj interface{}) (bool, interface{},
 		return true, &augmentedReview{AdmissionRequest: data.AdmissionRequest, Unstable: &unstable{Namespace: data.Namespace}}, nil
 	case *SideloadNamespace:
 		return true, &augmentedReview{AdmissionRequest: data.AdmissionRequest, Unstable: &unstable{Namespace: data.Namespace}}, nil
+	case unstructured.Unstructured:
+		admissionRequest, err := processAdmissionRequestFromUnstructured(data)
+		if err != nil {
+			return false, nil, err
+		}
+		return true, admissionRequest, nil
+	case *unstructured.Unstructured:
+		admissionRequest, err := processAdmissionRequestFromUnstructured(*data)
+		if err != nil {
+			return false, nil, err
+		}
+		return true, admissionRequest, nil
 	}
 	return false, nil, nil
 }
 
+func processAdmissionRequestFromUnstructured(obj unstructured.Unstructured) (admissionv1beta1.AdmissionRequest, error) {
+	resourceJSON, err := json.Marshal(obj.Object)
+	if err != nil {
+		return admissionv1beta1.AdmissionRequest{}, errors.New("Unable to marshal JSON encoding of object for audit")
+	}
+
+	req := admissionv1beta1.AdmissionRequest{
+		Kind: metav1.GroupVersionKind{
```

Comment:
For namespaced resources we should set the namespace info

Reply:
Fixed per our discussion, we will now pass from the audit an AugmentedUnstructured with the Namespace